extensions/amp-viewer-integration/0.1/amp-viewer-integration.js

```diff
@@ -72,7 +76,7 @@ export class AmpViewerIntegration {
 
     const ampdoc = getAmpDoc(this.win.document);
 
-    if (this.isWebView_) {
+    if (this.isWebView_ || this.isHandShakePoll_) {
```

Comment:
Does this mean that iframe poll always require port exchange? Is this an ok requirement per existing viewers?